pkg/environment/environment_test.go

```diff
@@ -15,13 +15,57 @@
 package environment
 
 import (
+	"os"
 	"path/filepath"
+	"strings"
 	"testing"
 )
 
+func TestMustGetKrewPaths_resolvesToHomeDir(t *testing.T) {
+	home := os.Getenv("USERPROFILE")
```

Comment:
I partially agree but I'll be adding an env-var override soon for this. That's when this will actually be useful.

Also it's an independent way of finding homedir, so nothing wrong with that :) It's a small test.

Reply:
Even though this is a small test, please don't test/recreate the homedir.Homedir() logic. We won't use this code even with an env-var override.